examples/mysql_to_kafka.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+#
+# Output Kafka events to the console from MySQL replication stream
+#
+
+from pymysqlreplication import BinLogStreamReader
+from pymysqlreplication.row_event import (
+    DeleteRowsEvent,
+    UpdateRowsEvent,
+    WriteRowsEvent,
+)
+
+from kafka.admin import KafkaAdminClient, NewTopic
+from kafka import KafkaConsumer, KafkaProducer
+
+MYSQL_SETTINGS = {"host": "127.0.0.1", "port": 3306, "user": "root", "passwd": ""}
+
+
+def create_kafka_topic(topic_name, num_partitions=1, replication_factor=1):
+    admin_client = KafkaAdminClient(bootstrap_servers="127.0.0.1:9092")
+
+    topic = NewTopic(
+        name=topic_name,
+        num_partitions=num_partitions,
+        replication_factor=replication_factor
+    )
+
+    admin_client.create_topics(new_topics=[topic])
+
+
+def main():
+    global message_body
+    producer = KafkaProducer(
+        bootstrap_servers='127.0.0.1:9092',
+        value_serializer=lambda v: str(v).encode('utf-8')
+    )
+
+    stream = BinLogStreamReader(
+        connection_settings=MYSQL_SETTINGS,
+        server_id=3,
+        only_events=[DeleteRowsEvent,UpdateRowsEvent,WriteRowsEvent]
+    )
+
+    for binlogevent in stream:
+
+        for row in binlogevent.rows:
+            if isinstance(binlogevent, DeleteRowsEvent):
+                topic = "deleted"
+                message_body = row["values"].items()
+
+            elif isinstance(binlogevent, UpdateRowsEvent):
+                topic = "updated"
+                message_body = row["after_values"].items()
+
+            elif isinstance(binlogevent, WriteRowsEvent):
+                topic = "created"
+                message_body = row["values"].items()
+
+            producer.send(topic, key=None, value=message_body)
```

Comment:
```suggestion
            producer.send(topic, key=None, value=dict(message_body))
```
I think wrapping dict or json format is more better.

Reply:
Yes. Current code might also impose some serialization issue. I will run some test thoroughly to prevent potential issues.

Reply:
I also think it's good for serialization. I wrap that message_body
I have one question.
In current code, I close stream, producer, consumer.
So I can only get the message before the main() execution.
Is that any problem?

Reply:
I think you did a good job on that one.
My expectations were that you would implementing only producing of binlogevents.
Yet, you managed to implement the whole package. 🙌🏻🙌🏻🙌🏻
I think your code is straightforward and it wouldn't be any problem at all.

I'll get back to you with the serialization issue after my test is done.